app/screens/thread/thread.js

```diff
@@ -96,10 +104,26 @@ export default class Thread extends PureComponent {
         }
     };
 
+    handleAutoComplete = (value) => {
+        if (this.postTextbox?.current) {
+            this.postTextbox.current.handleTextChange(value, true);
+        }
+    };
+
+    handleKeyboardHeightChange = (e) => {
+        if (e?.nativeEvent && this.keyboardLayout?.current) {
```

Comment:
Can `e` be falsey here? It seems like it should always be defined

Reply:
this is just a precaution to be safe..